### PR TITLE
AB#5150 Remove Lorem ipsums from Tracking

### PR DESCRIPTION
**Pull request by kevinh4**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js [hunk 1]
```diff
@@ -251,6 +251,7 @@ class RiskTrackingPopover extends Component {
       R.pathOr([], ['logged_by']),
       R.mergeAll,
     )(node);
+    console.log('Node', node)
```
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js:254]: Is this needed?

**Comment by kevinh4** [comment]
This PR can be checked and validated but wait until we're updated in staging before we merge.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js (hunk 1 above)
Comment (kevinh4) [reply opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js:254]: That should go away but I think we will need a ticket later to go clean up all of the console.logs

Comment (CyberDaedalus00) [reply opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js:254]: I'm ok with letting this go through IF a new ticket to clean up is created

**Pull request closed without merging**